core/src/main/java/com/hamdan/forzenbook/core/GlobalConstants.kt

```diff
@@ -2,5 +2,6 @@ package com.hamdan.forzenbook.core
 
 object GlobalConstants {
     const val EMAIL_LENGTH_LIMIT = 64
-    const val LOGIN_BASE_URL = "https://forzen.dev/api/"
+    const val LOGIN_BASE_URL = "https://forzen.dev/napi/"
```

Comment:
Not napi yet

Reply:
yea this was just left as napi from when we were testing with it. Changed back to api.